src/testperf.html

```diff
@@ -8,7 +8,7 @@
     -->
   </head>
   <body>
-      {% if request.user.is_staff %}
+      {% if request.user %}
```

Comment:
Should we put the same condition around the tab that links here?

Reply:
I think the UI is better this way given how often people forget to login. Instead of "I could have sworn such and such a feature exists, where is it?" I prefer "Oh...the site tells me I'm not logged in."

Reply:
Oh right, I was thinking this is a situation where the user just sees a raw 403, but since it's a condition within the page it's better than that.

Reply:
I fixed it and as penance for my mistake I also added tests.

Reply:
Looks like the test is failing here -- maybe some difference between template configuration between test/local/production settings?

Reply:
No good deed goes unpunished. 

I'd better disable the test now and add a ticket to look into this later. Its some interaction between build and test phases. The permission issue is blocking users and getting it enabled for our internal users is actually higher priority than keeping it secret from outsiders. The security is more of a nice to have to discourage DDOS attacks through slow queries. And I'm 95% sure it works now based on my desk-testing, whether the unit tests pass or not.

Reply:
Or to be more clear: the tests *do* pass on my laptop and my desk-checking passes but the tests don't run on CI due to a build/test interaction issue that I don't have time to investigate today.

Reply:
I'll merge this so we can get it deployed. I suspect the issue is that webpack is not running in the test setup, and that's how testperf.html gets generated.